classes/class-uagb-admin-helper.php

```diff
@@ -80,6 +85,7 @@ public static function get_admin_settings_shareable_data() {
 						'call-to-action',
 					)
 				),
+				'wp_is_block_theme'                 => $wp_is_block_theme,
```

Comment:
```suggestion
				'wp_is_block_theme'                 => function_exists( 'wp_is_block_theme' ) ? wp_is_block_theme() : false ,
```

If this is single time use can we do this instead of creating variable